common/lib/dependabot/experiments.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Dependabot
+  module Experiments
+    def self.register(name, value)
+      @experiments ||= {}
+      @experiments[name.to_sym] = value
+    end
+
+    def self.enabled?(name)
+      @experiments[name.to_sym]
```

Comment:
Would this be equivalent to `@experiments.key?(name)`?  It wouldn't let us pass a `false` value to the experiments, but I guess that's kind of pointless anyway?

Reply:
I think of `@experiments.key?(name)` as being a little surprising because you could be storing `false` or `nil` in the value. FWIW, I don't think this change is _necessary_, but the `true`/`false` from a predicate is something I think is worth sticking to.

Reply:
The same goes for what you're proposing though, right?

```ruby
irb(main):001:0> @experiments = { test: false }
=> {:test=>false}
irb(main):002:0> !@experiments[:test].nil?
=> true
```

Reply:
🤦‍♂️ Yeah, you're right. Maybe `!!@experiments[name]`?